### ISSUE TITLE
Weird styling on ubuntu 22.04

### ISSUE THREAD

**Issue by JoltCode** [johnlindquist/kit#1180]
As you can see below, when using script kit on ubuntu 22.04, there is weird styling which makes it very hard to read and use - I'm assuming this is not a bug?

![image](https://user-images.githubusercontent.com/46378904/229078254-7c1de79f-2de1-4d9f-98c2-406545fe87ff.png)

**Comment by johnlindquist** [comment]
@JoltCode 

Two things to try:

1. Disable opacity
In `~/.kit/db/theme.json`:

Change: `"--opacity":"0.75"` to `"--opacity":"1"`

2. Disable GPU
In `~/.kit/db/app.json`

1. Change:   `"disableGpu": false` to   `"disableGpu": true`
2. Quit Kit, then re-open (it can't apply the disableGpu change without it restarting)


Please let me know if either solution works for you. I rely pretty heavily on the community for testing Linux builds

**Comment by JoltCode** [comment]
Thank you! Will try 👍

**Comment by JoltCode** [comment]
Hey @johnlindquist,

![image](https://user-images.githubusercontent.com/46378904/235931225-3425126f-04aa-463f-8a8a-c1006cc075bb.png)
I've been trying to get it to run over the past few weeks, I've installed it using the npx command as well, but it seems to run but does not actually open.
![image](https://user-images.githubusercontent.com/46378904/235931547-d20f7937-f3b9-4e4d-bd67-3dcbb4d9926b.png)

Any ideas? Thanks!

**Comment by JoltCode** [comment]
Never mind I've managed to get it to work now with the ctrl+; - wlll let you know on the opacity things.

**Comment by JoltCode** [comment]
Yep your first suggestion has worked - changing the `"--opacity":"0.75"` to `"--opacity":"1"` has worked! Thanks :smile:
